code/game/objects/structures/ash_catcher.dm

```diff
@@ -0,0 +1,111 @@
+#define ASH_CATCHER_INTACT 0 //No ash caught.
+#define ASH_CATCHER_STRESSED 150 //The goliath hide is starting to buckle, but is still holding.
+#define ASH_CATCHER_COLLAPSING 300 //The hide can't hold much more before it falls apart.
+#define ASH_CATCHER_COLLAPSE 400 //The whole thing collapses!
+
+//Ash catchers are goliath-hide tarps that allow for expansion into lavaland without worrying about ash storms.
+//They accumulate ash over time and must be periodically shaken out.
+//If the catchers are too heavy and collect too much ash, they will collapse and allow ash storms through.
+/obj/structure/ash_catcher
+	name = "ash catcher"
+	desc = "A sheet of goliath's hide stretched over four metal rods."
```

Comment:
I kept it out of the description because it might be too spooky for people and I can't add a spook warning for everything.

(I'm gonna add a subtype in the next commit, don't worry!)